extensions/elytron-security/deployment/src/test/java/io/quarkus/security/test/jdbc/ParametrizedPathsResource.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.security.test.jdbc;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+
+/**
+ * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
+ *         <br>
+ *         Date: 6/13/18
+ */
+@Path("/parameterized-paths")
+public class ParametrizedPathsResource {
```

Comment:
Why don't you reuse the existing `io.quarkus.security.test.ParametrizedPathsResource` ?